src/sql/platform/restore/browser/restoreServiceImpl.ts

```diff
@@ -165,11 +166,10 @@ export class RestoreDialogController implements IRestoreDialogController {
 			const self = this;
 			let connectionProfile = self._connectionService.getConnectionProfile(self._ownerUri);
 			let activeNode = self._objectExplorerService.getObjectExplorerNode(connectionProfile);
-			this._taskService.onTaskComplete(response => {
+			this._taskService.onTaskComplete(async response => {
```

Comment:
this isn't doing anything, just hiding the error. since the event emitter doesn't care about the return type of the handlers. Instead you should just catch the errors within the handler.

Reply:
Sounds good!